cache/cache_test.go

```diff
@@ -178,18 +178,18 @@ func testAdd(t *testing.T, newCache cacheFactory) {
 	var err error
 	cache := newCache(t, time.Hour)
 	// Add to an empty cache.
-	if err = cache.Add("int", 1, time.Second); err != nil {
+	if err = cache.Add("int", 1, time.Second * 3); err != nil {
 		t.Errorf("Unexpected error adding to empty cache: %s", err)
 	}
 
 	// Try to add again. (fail)
-	if err = cache.Add("int", 2, time.Second); err != ErrNotStored {
+	if err = cache.Add("int", 2, time.Second * 3); err != ErrNotStored {
```

Comment:
Somehow travis-ci produced [a failure](https://travis-ci.org/revel/revel/jobs/104316303#L205).
Cache TTL looks too short so I make it longer.